### PR TITLE
KFLUXINFRA-1131: Overlays for kflux-prd-rh02

### PR DESCRIPTION
**Pull request by gbenhaim**
Create the overlays for:

- build-service
- mintmaker
- pipelines-service
- integration-service
- monitoring

console urls for the pipeline service and integration will be added after the cluster will be ready.

### CODE REVIEW

File: components/build-service/production/kflux-prd-rh02/kustomization.yaml
Comment (hugares) [inline components/build-service/production/kflux-prd-rh02/kustomization.yaml:1]: For all the components with specific folder, you need to change the application set to add the cluster name in the list, e.g.: https://github.com/redhat-appstudio/infra-deployments/blob/main/argo-cd-apps/base/member/infra-deployments/build-service/build-service.yaml#L25

Comment (gbenhaim) [reply components/build-service/production/kflux-prd-rh02/kustomization.yaml:1]: I know. I  planned to handle the argo manifests in a different pr.

Comment (hugares) [reply components/build-service/production/kflux-prd-rh02/kustomization.yaml:1]: Ack

**Review by hugares — APPROVED** [review]
/lgtm

**Pull request merged**